governance/default.go

```diff
@@ -608,17 +608,18 @@ func newGovernanceCache() common.Cache {
 
 func (g *Governance) initializeCache() error {
 	// get last n governance change block number
-	indices, err := g.db.ReadRecentGovernanceIdx(params.GovernanceCacheLimit)
+	indices, err := g.db.ReadRecentGovernanceIdx(params.GovernanceIdxCacheLimit)
```

Comment:
This has been changed from `GovernanceCacheLimit` to `GovernanceIdxCacheLimit`. Is it intended?
Since the one value changed is `GovernanceCacheLimit`

Reply:
@ehnuje Yes. It is intended. Initially, the number of `GovernanceCacheLimit` was very small. It seemed enough to keep up with block increase, it couldn't respond well for a random search like an API call. Jun insisted to increase to hold all in cache. Keeping all governance configuration in memory is worrying because of memory usage, but keeping just index won't take that amount of memory. So I added a new constant `GovernanceIdxCacheLimit` which has twice bigger than `GovernanceCacheLimit` which limits the number of ItemCache